lib/elm-test.js

```diff
@@ -103,13 +101,18 @@ if (args.compiler === undefined) {
 (If elm is not on your path or is called something different the --compiler flag might help.)`
     );
     process.exit(1);
+    // Flow does not understand that `process.exit()` diverges, we add an
+    // unconditional throw here (that can never run) to help flow out.
+    throw new Error();
```

Comment:
This was my initial solution too, but the annoying thing is that VSCode then marks this line as not reachable, which is noisy and tempts me to remove it all the time.

Reply:
Came up with a best of both worlds solution: [`ef41189` (#448)](https://github.com/rtfeldman/node-test-runner/pull/448/commits/ef41189c571b140d2d952f60630c5f6405f07abb)